### PR TITLE
brand domain based on deploy type

### PR DESCRIPTION
**Pull request by wraeth-eth**
### Description

- Brand juicecrowd onto juicecrowd projects, else juicebox in metadata

### CODE REVIEW

**Comment by wraeth-eth** [comment]
Current dependencies on/for this PR:
* main
  * **PR #4144** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/4144?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/4144?utm_source=stack-comment).

File: src/components/Create/hooks/DeployProject/useDeployProject.ts [hunk 1]
```diff
@@ -219,13 +223,22 @@ export const useDeployProject = () => {
         } else {
           deleteIfExists(projectMetadata, 'softTargetAmount')
         }
+
+        // Set domain to juicecrowd if juicecrowd project
+        if (
+          projectMetadata.softTargetAmount &&
+          (projectMetadata.introVideoUrl || projectMetadata.introImageUri)
+        ) {
+          domain = JUICECROWD_DOMAIN
```
Comment (tomquirk) [inline src/components/Create/hooks/DeployProject/useDeployProject.ts:232]: **suggestion**: could just use the feature flag to set the defaultState

Comment (wraeth-eth) [reply src/components/Create/hooks/DeployProject/useDeployProject.ts:232]: Thought about that, but theres a chance someone forgets. it feels unexpected to do this

**Review by tomquirk — APPROVED** [review]

**Comment by wraeth-eth** [comment]
### Merge activity

* **Nov 18, 5:32 PM**: @wraeth-eth started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/4144).

**Pull request merged**